### ISSUE TITLE
Description for Sync/Playback options unclear

### ISSUE THREAD

**Issue by thenktor** [MediaBrowser/plugin.video.emby#25]
In Sync options the description for playback mode "Add-on" says: 
> This playback mode is controlled by the add-on. You can direct play/direct stream/transcode and use the Emby cinema feature.

While the description for playback mode "Native" mentions: 
> This will use actual file paths to your media files. These paths come from your Emby server. Make sure the Kodi client has access to the same paths used by Emby, or use Path Substitution in the Library options on the server.

This leaves the impression that when selecting "Add-on" you do not have to mount any paths on the client and things will just work (which was not the case for my test).

But then in Playback options there is the setting "Play from HTTP instead of SMB" with description: 
> Enable this option to direct stream/transcode.

Basically that sounds the same like the "Add-on" playback mode. Or at least "Add-on" mode does not make sense without HTTP streaming for me.

**Comment by angelblue05** [comment]
I understand. I'll check to see what we can do to clarify this. Play from http instead of SMB only work with addon playback. It's just to force it to stream rather than try to direct play first (if you are outside your network, for example).

To direct play (regardless of the playback mode), you always need to make sure your path are accessible from the client. This means, making sure your library paths are network shares rather than local paths.

**Comment by thenktor** [comment]
> Play from http instead of SMB only work with addon playback.

So it probably should be a sub-option of Addon playback?

I guess there also is at least one bug. I've set playback mode to Addon and did not mount any share. In this case it could not play a file at all. IIRC in the log appeared access to the file path first (which of course failed) and afterwards to a http stream (which also failed). Mounting the share and setting up path substitution made it work.
Yesterday I've unmountes the share and enabled http playback, which also worked. But still in the GUI the "opening stream" message appeared twice, which makes me think that the share still is accessed first (did not check the log, sorry).

**Comment by angelblue05** [comment]
> So it probably should be a sub-option of Addon playback?

No, the playback mode is a sync option, while the play from http is a playback option. They go together, as in, for the add-on to control what happens before playback, you need to use addon playback. So for you to be able to go from directplay to direct stream/transcode, you need to use addon playback. If you use native playback and you want the option, you'd have to resync your entire library to change the playback mode. Native mode is only for direct play, Kodi controls everything.

Without a log, it's impossible to tell what's going on. Feel free to open a thread and post your kodi log here: http://emby.media/community/index.php?/forum/99-kodi/ as we don't use github to help with troubleshooting (In other words, I don't see your new replies).
